docker-entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -e
```

Comment:
This file shouldn't be committed!

Reply:
Sorry... a serious error of negligence, it will not happen again!